packages/core/package.json

```diff
@@ -49,6 +49,7 @@
     "web-did-resolver": "^2.0.8"
   },
   "devDependencies": {
+    "node-fetch": "^3.2.6",
```

Comment:
node-fetch 3 uses esm only, so we should probably stick to 2.x (the version that is used in the @aries-framework/node package)